app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::MessagesController < ApplicationController
+  def index
+    @messages = Message.all
+    render json: @messages
+  end
```

Comment:
- [ ] After fixing the issue above, you still need to configure your backend and the frontend to display a **random** greeting, not all greetings. Every time the front end is refreshed the API should return a random message and the Frontend should display it. For example, you can change this method to the code below and refactor the FE as well to handle it.

  ```
  def index
    message = Message.all.sample
    render json: { message: message }
  end
  ``` 

  ![fjfk](https://user-images.githubusercontent.com/36542986/196938268-c1c71b1e-9e57-4e96-a33e-435ede1d3a93.jpg)